### ISSUE TITLE
Idle doesn't work

### ISSUE THREAD

**Issue by AndreiMihalea** [pop-os/cosmic-idle#13]
Hello,

I've tried modifying power options to turn off the screen after two minutes and nothing happens. I left my monitors on for hours and they never turned off. Do I need to run cosmic-idle in terminal, or is it enough to just set up things in power settings?

Also, is there a theoretical possibility to immediately shut down the monitors after locking the computer?

**Comment by ids1024** [comment]
`cosmic-session` should start `cosmic-idle` automatically.

**Comment by AndreiMihalea** [comment]
Yes, that was my bad. After updating to the latest version of cosmic-session, it now works.
Would still be nice to have that feature with shutting off the displays when locking, without having to wait for the idle time.
